jvmti/src/cpp/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 FILE(GLOB Source *.cc)
 add_library(jvmti SHARED ${Source})
-set_target_properties(jvmti PROPERTIES SUFFIX ".so")
+
+# set static lib suffix
+if(WIN32)
+    set_target_properties(jvmti PROPERTIES SUFFIX ".dll")
```

Comment:
Is it possible to have SUFFIX always ".so"? 

I'm using this hack to simplify the process attaching the agentlib: https://github.com/cmu-pasta/fray/blob/main/plugins/gradle/src/main/kotlin/FrayPlugin.kt#L59